src/Operations/ConvertContoursToMeshes.cc

```diff
@@ -202,7 +202,14 @@ bool ConvertContoursToMeshes(Drover &DICOM_data,
     if(std::regex_match(MethodStr, direct_regex)){
         // Identify the unique planes.
         //const auto est_cont_normal = cc.contours.front().Estimate_Planar_Normal();
-        const auto est_cont_normal = Average_Contour_Normals(cc_ref);
+        const auto est_cont_normal = [&]() {
+            auto est_cont_normal = Average_Contour_Normals(cc_ref);
+            if(!est_cont_normal.isfinite()) {
+                est_cont_normal = cc_ref.front().get().contours.front().Estimate_Planar_Normal();
+            };
+            return est_cont_normal;
+        }();
```

Comment:
Does this address the symptom or the cause of this issue? Here we're generating the contours, and from my understanding the issue is that this exact alignment means normalization results in a divide by 0. Just to confirm though, if there is a slight offset there won't be any large-but-finite values returned that this wouldn't catch, right?

Also I'm very much not a fan of getting clever with lambdas like this unless there's a reason I'm missing for it. Why couldn't you just remove the lambda wrapper like this? 
```
auto est_cont_normal = Average_Contour_Normals(cc_ref);
if(!est_cont_normal.isfinite()) {
   est_cont_normal = cc_ref.front().get().contours.front().Estimate_Planar_Normal();
};
```

Reply:
Addresses symptom. Cause is due to two planes having a normal of (0,0,1), and one plane having a normal of (0,0,-1). Could fix cause in ContourViaThreshold but would involve changing how we process images to contours. I can change the code to what you suggested, only reason for a lambda function was to allow est_cont_normal to be constant, as it was before.